pkg/httpserver/httpserver_test.go

```diff
@@ -144,3 +113,49 @@ var _ = Describe("Test httpserver", func() {
 		Expect(resp.StatusCode).To(Equal(http.StatusOK))
 	})
 })
+
+var _ = Describe("Test debugserver", func() {
```

Comment:
We're trying not to use the Ginkgo framework for testing. It'd be great as a follow up to convert these to tests that use the standard testing package while you still if context and if you have bandwidth.